http/file_server.ts

```diff
@@ -806,21 +806,46 @@ function main() {
 
   const useTls = !!(keyFile && certFile);
 
+  function onListen({ port, hostname }: { port: number; hostname: string }) {
+    const networkAddress = getNetworkAddress();
+    const protocol = useTls ? "https" : "http";
+    let message = `Listening on:\n- Local: ${protocol}://${hostname}:${port}`;
+    if (networkAddress) {
```

Comment:
```suggestion
    if (networkAddress && DENO_DEPLOYMENT_ID === undefined) {
```
So nothing is printed in Deno Deploy.